front/app/utils/moduleUtils.tsx

```diff
@@ -601,7 +601,7 @@ export const insertConfiguration =
       // if number is outside of lower and upper, it picks
       // the closes value. If it's inside the ranges, the
       // number is kept
-      insertAfterName ? referenceIndex + 1 : referenceIndex - 1,
+      insertAfterName ? referenceIndex + 1 : referenceIndex,
```

Comment:
Can you explain this change? I've never seen this function before, it looks kind of complicated

Reply:
Indeed, this function is complicated; maybe we could simplify it in the future. I hope this commit msg can explain a bit more https://github.com/CitizenLabDotCo/citizenlab/pull/2428/commits/ea13185df8241da374e47493aec2945797f5afd3

Basically, in this function, we insert a config into an array of configs. We insert after the specified name (`insertAfterName` is passed) or before the specified name (`insertBeforeName` is passed). So, in case of `insertBeforeName`, the config was inserted not right before but two positions before. E.g., we have an array [1,2,3,4,5], and we have to insert 6 before 4. Before the fix, it was inserted between 2 and 3. And now it's inserted between 3 and 4.

Reply:
Solid, thanks for the explanation. PR looks good 🎢